core/local/atom/add_checksum.js

```diff
@@ -60,3 +59,13 @@ function loop(
     return events
   })
 }
+
+function isFileWithContent(event /*: AtomEvent */) /*: boolean %checks */ {
+  return (
+    (event.action === 'created' ||
+      event.action === 'modified' ||
+      event.action === 'scan' ||
+      event.action === 'renamed') &&
+    event.kind === 'file'
```

Comment:
Maybe use a `Set`?

```js
const contentActions = new Set(['created', 'modified', 'renamed', 'scan'])
const isFileWithContent = ({ action, kind }) => kind === 'file' && contentActions.has(action)
```

Reply:
This method is a Flow predicate. Its body needs to be a single return expression for Flow to understand it.